Makefile.jenkins

```diff
@@ -86,6 +95,7 @@ ui-test:
 	CI=true yarn test --maxWorkers=4
 
 ui-npm-publish:
+ifeq ($(PUSH_TO_NPM), y)
```

Comment:
Did you try that the condition works as expected?
Sometimes I had problems when using spaces inside the parenthesis, because `make` tries to compare a "trimmed" string with an non-trimmed string.

I see that the NPM_DRY_RUN condition also has the space, so just asking to be sure.

Reply:
It works, I tested this :)